test/e2e/atlas/kubernetes_config_apply_test.go

```diff
@@ -228,6 +228,8 @@ func TestKubernetesConfigApply(t *testing.T) {
 				true,
 			),
 		)
+		assert.NotEmpty(t, akoDeployment.Spec.DeploymentSpec.MongoDBMajorVersion)
+		akoDeployment.Spec.DeploymentSpec.MongoDBMajorVersion = ""
```

Comment:
Why not add the expected value to the expected resource instead of cleaning up the field?

Reply:
Since we don't set the specific version for the test, we don't know what version is returned by the API

Reply:
Got you, in that case it's on Atlas side the decision and it set to the default version. What about rely on something like this which is used by the cluster commands https://github.com/mongodb/mongodb-atlas-cli/blob/1cf356959e1af535d8b23a9dc327639d0f616bed/internal/cli/atlas_cluster_opts.go#L24

Reply:
Good point. Fixed!